debian/netscanner.manpage

```diff
@@ -0,0 +1,15 @@
+.\" Manpage for netscanner.
+.\" Contact dominic.orsi@gmail.com to correct errors or typos.
+.TH man 1 "22 June 2024" "1.0" "netscanner man page"
+.SH NAME
+netscanner \- view and scan connected networks
```

Comment:
May I please ask you to change it to general description: `Network scanner and diagnostic tool` ?

Reply:
Oh, now I see that we have description under `NAME`. So this is just a addition to the name itself, right ?

Reply:
Yes, it is just an addition to the name itself. See the screenshot for how the manpages is laid out. 
![Netscanner-manpage](https://github.com/Chleba/netscanner/assets/73092665/37e67f0e-e359-48b2-ae9b-c2b3d1e60370)

Reply:
See commit [5a30139](https://github.com/Chleba/netscanner/pull/33/commits/5a301395f3004340d61affffcb9fb3c3f8d3cb66)